test/features/support/oidc_introspect.js

```diff
@@ -0,0 +1,132 @@
+const { spec, stash } = require("pactum");
+const { Given, When, Then, Before, After } = require("@cucumber/cucumber");
+const { localhost } = require("./helpers/helpers");
+
+const baseUrl = `${localhost}introspect`;
+
+let specIntrospect;
+
+stash.addDataTemplate({
+  QueryParams: {
+    client_id: "string",
+    token: "string.string.string",
+    token_type_hint: "access_token",
+  },
+});
+
+stash.addDataTemplate({
+  ResponseSchemaOK: {
+    type: "object",
+    properties: {
+      active: { type: "boolean" },
+    },
+    required: ["active"],
+    additionalProperties: true,
```

Comment:
'additionalProperties' is equal to 'true' by default, so you can remove it :) But should we allow additional properties?

Reply:
I expect we should when the token comes as active.
![image](https://user-images.githubusercontent.com/119664031/216037116-7a1b78f6-9810-48e8-ae77-e5663c6485d1.png)

Reply:
Ok, do you think you can remove this line so? By default 'additionalProperties' is equal to 'true'

Reply:
Yes. Done.